### PR TITLE
feat(blog): add guest post by "Do you (really) need more?"

### PR DESCRIPTION
**Pull request by real34**
Hi!

As discussed, here is a blog post where I tried to explore how OpenMage could fit for merchants in the current online commerce ecosystem.

Please don't hesitate to let me know any changes that might feel needed, or if you want us to add a word about Front-Commerce somewhere (footer?).

_I mark it as a Draft because I still need to figure out a thumbnail that could fit :smile:_

### CODE REVIEW

**Review by Flyingmana — APPROVED** [review]

**Comment by Flyingmana** [comment]
I think the mention in the opener is already good enough to make it transparent. But if you want, you can also add some more words about it in the end.

Thank you a lot for this blog post.

**Comment by fballiano** [comment]
I can't oppose to anything but I will state that I am totally against sponsoring blog posts.

**Comment by Flyingmana** [comment]
@fballiano wording is important here, its not a sponsored blog post. Its a guest blog post.
The only thing we get from this is the blog post, and possible traffic of people reading it.
This is an important differentiation because Search Engines have very strict rules about sponsored content, and describing it falsely as such, could cause us a lot of damage.

**Comment by fballiano** [comment]
I said sponsoring, not sponsored. And to me these posts causes more damage that my opinion.

**Comment by real34** [comment]
Hi!

I finally found time for updating the PR, it should be ready to go.

I've seen recent discussions about the OM future and contributions, and I don't want to interfere with such discussion since we're not actively contributing to the core. This guest post was indeed discussed months ago and I procrastinated to write it. Since then vision and opinions in the core team can have changed, and @fballiano already mentioned additional interesting questions about the site in https://github.com/OpenMage/magento-lts/discussions/2873#discussioncomment-4570506.

Our goal here was to contribute in our way, by explaining what our vision was when we started to follow the OM initiative and discuss with merchants still on M1 about their future. OM recent improvements have proven us right here, and over the years we've seen many use cases around headless usage … so we thought it would make sense to contribute with content.

We still meet merchants with an existing stores who would not benefit from a migration to other systems, and we still think that the OM is relevant for them. That was why we thought it could be good to have such explanations published here, as going headless is usually a good compromise for them.

So far we can only contribute with content and communication around headless use cases, even though I'd personally love to do more. However, if you don't think it should go on this website we could publish it on ours and close this PR.

**Review by colinmollenhour — COMMENTED** [review]
Thanks for contributing this blog post, @real34 !

I made some edit suggestions. For now the official name of the code-base is "OpenMage LTS" although that could change in the future.

@fballiano Yes, the blog post contains some not-so-subtle hat-tips to the Front Commerce business, but it is good for SEO to have fresh original content and it is arguably a very appropos topic for our users and makes good points so I think it is a worthwhile read for those considering sticking with Magento via OpenMage vs. replatforming. Even if such partnerships do not involve active contributions to the OM code, as long as their product is actively supporting a technical interoperation with our code base, this constitutes a relationship worthy of recognition (IMHO).

File: _posts/2023-01-03-do-you-really-need-more.md
Comment (real34) [reply _posts/2023-01-03-do-you-really-need-more.md:83]: Good catch!\
I wanted some line breaks, but Jekyll's markdown parser uses double spaces `  ` instead of `\` … and my editor autoformat removed them without me noticing.

I'll replace with `<br>` tags:
```suggestion
Do you want to grow internationally? Support new payment and shipping methods used by customers from different countries.<br>
Do you want to sell on a marketplace? Export your catalog to the marketplace your targeted customers visit daily.<br>
Do you want to engage your community? Embed widgets from your main social media platforms on your website, and optimize your pages for sharing and reposting.
```

**Comment by fballiano** [comment]
@real34 I'm not in any position to decide anything, I'm just expressing my opinion here.

@colinmollenhour I think the benefit for the partner companies that are featured on the website is much larger than the other way around. we would need help, in form of manpower mainly, and we're providing exposure in exchange for nothing. Most of the companies listed as partners are not even updating their modules or talking about OpenMage in their services, that is not fair to OM or to the people reading the partner companies page.

**Comment by colinmollenhour** [comment]
@fballiano This should maybe be discussed more privately but there are other possibilities like FC allowing us to cross-post on their blog (but then we need someone to write a post), or promoting OpenMage in a targeted newsletter to Magento 1 users or something of that nature.

@real34 Just out of curiosity, can you share some metrics or anecdotes on how much adoption your Magento 1 integration has? I know you specifically metnion support of OpenMage on the M1 installation page already. 👍

**Review by colinmollenhour — APPROVED** [review]
Thanks for the info @real34!
@Flyingmana I have no objection to merging, I'll leave it up to you as to if this needs any further discussion but it seems as it was already agreed upon before the post was written and the post has now been reviewed and edited we should accept it and perhaps circle back regarding the broader discussion elsewhere/later.

**Comment by real34** [comment]
Thanks everyone. Again, feel free to come back to me if you end up deciding such posts are not welcome on the site and we won't be disappointed or frustrated to relocate it on our blog instead.

We're also available to discuss expectations you'd love from partners and endorse this responsibility more, with our capacity (publishing content, demos, contributions or feedbacks).

**Pull request merged**
